doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1100,6 +1100,14 @@ Other API Changes
 Deprecations
 ~~~~~~~~~~~~
 
+- :attr:`MultiIndex.labels` has been deprecated and replaced by :attr:`MultiIndex.codes`.
+  The functionality is unchanged. The new name better reflects the natures of
+  these codes and makes the ``MultiIndex`` API more similar to the API for
+  :class:`CategoricalIndex`(:issue:`13443`).
```

Comment:
this might be a doc-rendering problem, but can resolve later

Reply:
Not sure I understand.